### PR TITLE
Ch1 edits

### PR DESCRIPTION
**Pull request by echalmers**
Just necessary corrections and wording fixes for chapter 1. I suspect there will be more edits needed to later chapters, but let me know if you were hoping for a higher level of editing & revision.
Some of the presentation isn't how I would have done it, but it's a free textbook right? :)

### CODE REVIEW

**Comment by PJPerri** [comment]
﻿﻿EricWe cool but - I thought I had the folders from Functions on (meaning General Info was mine to do. PJ PerriSenior InstructorMount Royal UniversityHe/His/HimOn Jun 14, 2023, at 4:18 PM, echalmers ***@***.***> wrote:﻿
@echalmers requested your review on: #6 Ch1 edits.

**Comment by PJPerri** [comment]
﻿﻿EricWe cool but - I thought I had the folders from Functions on (meaning General Info was mine to do. )PJ PerriSenior InstructorMount Royal UniversityHe/His/HimOn Jun 14, 2023, at 4:18 PM, echalmers ***@***.***> wrote:﻿
@echalmers requested your review on: #6 Ch1 edits.

**Comment by echalmers** [comment]
Ah! Right, thanks. Somehow I started thinking I was doing the first half chronologically instead of alphabetically.
Now that it's here - were you planning on doing more changes than this @PJPerri ?

**Comment by PJPerri** [comment]
EricTomorrow I will look and see. PJ PerriMRU Senior Lecturer, He/him/hisOki /Amba'wastitch / Danit'ada / Tans'iI send this email from the hereditary homelands of the Niitsitapi (the Blackfoot Confederacy: Siksika, Piikani, Kainai), the Îyârhe Nakoda, and Tsuut'ina Nations. Calgary is also home to the Metis Nation Region 3. We are all Treaty People. This communication is intended for the use of the recipient to which it is addressed, and may contain confidential, personal, and or privileged information. Please contact the sender immediately if you are not the intended recipient of this communication, and do not copy, distribute, or take action relying on it. Any communication received in error, or subsequent reply, should be deleted or destroyed.On Jun 14, 2023, at 4:44 PM, echalmers ***@***.***> wrote:﻿
Ah! Right, thanks. Somehow I started thinking I was doing the first half chronologically instead of alphabetically.
Now that it's here - were you planning on doing more changes than this @PJPerri ?

File: pretext/GeneralIntro/Algorithms.ptx [hunk 1]
```diff
@@ -34,7 +34,7 @@
           <p>Ask for radius</p>
         </li>
         <li>
-          <p>let area = (radius<superscript>2</superscript>) &#xD7; &#x3C0;</p>
+          <p>let area = <m>\pi \times r^2</m></p>
```
Comment (PJPerri) [inline pretext/GeneralIntro/Algorithms.ptx:37]: I too say that there is a <M> and <me> tag taht does math better

**Pull request merged**